lib/fastlane/actions/resize_app_icon.rb

```diff
@@ -0,0 +1,120 @@
+module Fastlane
+  module Actions
+    # ResizeAction is based on work created and Copyright (c) 2014 Erik Sundin
+    # Original source may be found under the MIT license at https://github.com/eriksundin/appicon
+    class ResizeAppIconAction < Action
+      def self.run(params)
+        # fastlane will take care of reading in the parameter and fetching the environment variable:
+        Helper.log.info "Path to source icon: #{params[:iconPath]}"
+        Helper.log.info "Path to Images.xcassets: #{params[:assetsPath]}"
+
+        if params[:iconPath] and params[:assetsPath] and !params[:iconPath].empty? and !params[:assetsPath].empty?
+          validate_image_magick!
+
+          determine_icon! unless @icon = params[:iconPath]
+          determine_asset_catalog! unless @asset_catalog = params[:assetsPath]
```

Comment:
Doh. They can be removed. In the original script they were used to ask for parameters if they weren't passed in on the original call.